### PR TITLE
feat(pgdialect): add identity support

### LINKED ISSUE #479: struct tags "autoincrement"
## PostgreSQL

[Don't use serial](https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_serial)

> For new applications, **identity columns** should be used instead.

### PR DESCRIPTION
**Pull request by oGi4i**
Closes #479

### CODE REVIEW

**Comment by oGi4i** [comment]
@vmihailenco thanks for your thoughts on this, please check out this implementation

File: query_table_create.go [hunk 1]
```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/uptrace/bun/schema"
 )
 
+type IdentityAppender interface {
+	AppendIdentity(b []byte) []byte
+}
```
Comment (vmihailenco) [inline query_table_create.go:17]: Seems to be unused. Let's remove.

Comment (oGi4i) [reply query_table_create.go]: Yeah, sorry, forgot to remove that. Done.

**Comment by oGi4i** [comment]
> @oGi4i looks good 👍 Could you also add a test to https://github.com/uptrace/bun/blob/master/internal/dbtest/query_test.go#L899 so we are sure the generated query stays the same?

Thanks.
Added a test case to check that create table query is generated correctly for all dialects

**Comment by vmihailenco** [comment]
Thanks :+1:

**Pull request merged**